packages/core-accountsjs/core-accountsjs.js

```diff
@@ -41,37 +41,44 @@ export const dbManager = new DatabaseManager({
 const accountsServerOptions = {
   siteUrl: process.env.ROOT_URL,
   prepareMail: (to, token, user, pathFragment, emailTemplate, from) => {
-    const actionsSet = {
-      'verify-email': 'verifyEmail',
-      'enroll-account': 'enrollAccount',
-      'reset-password': 'resetPassword',
-    };
-
-    return {
-      recipientEmail: to,
-      action: actionsSet[pathFragment],
-      userId: user.id || user._id,
-      token,
-    };
-  },
-  sendMail: ({ action, userId, token, recipientEmail }) => {
-    return WorkerDirector.addWork({
-      type: 'MESSAGE',
-      retries: 0,
-      input: {
-        template: 'ACCOUNT_ACTION',
-        action,
-        recipientEmail,
-        userId,
+    if (token && pathFragment) {
+      const actionsSet = {
+        'verify-email': 'verifyEmail',
+        'enroll-account': 'enrollAccount',
+        'reset-password': 'resetPassword',
+      };
+
+      return {
+        recipientEmail: to,
+        action: actionsSet[pathFragment],
+        userId: user.id || user._id,
         token,
-      },
-    });
+      };
+    }
+  },
+  sendMail: (data) => {
+    if (data) {
+      const { action, userId, token, recipientEmail } = data;
+      return WorkerDirector.addWork({
+        type: 'MESSAGE',
+        retries: 0,
+        input: {
+          template: 'ACCOUNT_ACTION',
+          action,
+          recipientEmail,
+          userId,
+          token,
+        },
+      });
+    }
   },
 };
 
 class UnchainedAccountsPassword extends AccountsPassword {}
 
-export const accountsPassword = new UnchainedAccountsPassword({});
+export const accountsPassword = new UnchainedAccountsPassword({
+  sendVerificationEmailAfterSignup: true,
```

Comment:
This has to be configurable from the outside, I think it was before by using "skipVerificationEmail" or something like that, else LGTM

Reply:
See an example on how to achieve "module" configurability:

https://docs.unchained.shop/config/assortments/

Technically:

https://github.com/unchainedshop/unchained/blob/master/packages/core-assortments/settings.js
-> settings provide a default value for everything

https://github.com/unchainedshop/unchained/blob/master/packages/core-assortments/assortments.js -> settings.load...

https://github.com/unchainedshop/unchained/blob/master/packages/core-assortments/db/helpers.js
-> importing settings

Reply:
You'd fine out that `accountsPassword` is accessed to set the variable internally, that's because it we tried to read the value off of settings like so `export const accountsPassword = new UnchainedAccountsPassword({
  sendVerificationEmailAfterSignup: settings.sendVerificationEmailAfterSignup,
});
` it'd fail to reevaluate after the configs are set. 

```
import { accountsPassword } from './accounts-password';

const sendVerificationEmail = true;

const settings = {
  load({ sendVerificationEmailAfterSignup = sendVerificationEmail } = {}) {
    accountsPassword.options.sendVerificationEmailAfterSignup = sendVerificationEmailAfterSignup;
  },
};
```

Reply:
Good solution in that case I guess :)